### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception *** exception class was changed or removed startRefreshLocationTimerAsync()

### ISSUE THREAD

**Issue by jj-triff-kr** [microsoft/azure-tools-for-java#7159]
Unable to refresh database account from any location. Exception: {"ClassName":"CosmosException","userAgent":"azsdk-java-cosmos/4.38.0 Windows10/10.0 JRE/17.0.5","statusCode":403,"resourceAddress":"https://cosmos-kr-fdp-sb-snd-eus2-cm-eastus2.documents.azure.com:443/","error":"{\"code\":\"Forbidden\",\"message\":\"Request originated from IP 136.226.59.34 through public internet. This is blocked by your Cosmos DB account firewall settings. More info: https://aka.ms/cosmosdb-tsg-forbidden\\r\\nActivityId: 47ce75db-f6ac-4bcb-9c7e-6994d2cd6962, Microsoft.Azure.Documents.Common/2.14.0, StatusCode: Forbidden\",\"additionalErrorInfo\":null}","innerErrorMessage":"Request originated from IP 136.226.59.34 through public internet. This is blocked by your Cosmos DB account firewall settings. More info: https://aka.ms/cosmosdb-tsg-forbidden\r\nActivityId: 47ce75db-f6ac-4bcb-9c7e-6994d2cd6962, Microsoft.Azure.Documents.Common/2.14.0, StatusCode: Forbidden","causeInfo":null,"responseHeaders":"{Transfer-Encoding=chunked, Strict-Transport-Security=max-age=31536000, Server=Microsoft-HTTPAPI/2.0, Content-Location=https://cosmos-kr-fdp-sb-snd-eus2-cm-eastus2.documents.azure.com/, x-ms-ga

**Comment by wangmingliang-ms** [comment]
Duplicate of #7898
